backend/mora/service/orgunit.py

```diff
@@ -158,8 +162,9 @@ def edit(self, unitid, req):
         payload = common.ensure_bounds(new_from, new_to, bounds_fields,
                                        original, payload)
 
-        validator.is_date_range_in_org_unit_range(parent_uuid, new_from,
-                                                  new_to)
+        if org_uuid != parent_uuid:
+            validator.is_date_range_in_org_unit_range(parent_uuid, new_from,
+                                                      new_to)
```

Comment:
We should probably validate that we're not exceeding the organisation's validity, shouldn't we?

Reply:
I think organisations are always just created to last forever, if we're suddenly outside the validity period of the organisation, we have bigger problems.

Reply:
Well, that depends on the importer — for example, most municipalities have a from day one might use. A simple `TODO` might suffice?